sources/web/datalab/polymer/components/directory-picker-dialog/directory-picker-dialog.ts

```diff
@@ -81,6 +92,21 @@ class DirectoryPickerDialogElement extends BaseDialogElement {
     return this._memoizedTemplate;
   }
 
+  ready() {
+    if (!this.readyPromise) {
+      super.ready();
+      this.readyPromise = this.$.filePicker.ready();
+      this.addEventListener('iron-overlay-opened',
```

Comment:
A comment will be very useful here about why we're doing this.

Reply:
Done.